PyTechBrain3_test

wrata opened the pull request:
Program testujący płytkę PyTechBrain z oprogramowaniem FirmataPlus.

wrata committed:
Add files via upload

Dodałem interfejs pinów analogowych.

wrata committed:
Add files via upload

Zmiany w interfejsie, usunięty odczyt pinów analogowych A0 i A1.

wrata committed:
Add files via upload

Wersja programu testującego płytkę PyTechBrain. Do uruchomienia potrzebny jest Python 3.5.1 lub nowszy, zainstalowany moduł pymata-aio (https://github.com/MrYsLab/pymata-aio) oraz PyTechBrain z zainstalowanym oprogramowaniem FirmataPlus (https://github.com/MrYsLab/pymata-aio/tree/master/FirmataPlus).

abixadamj commented:
Wiesiek, musze to ręcznie rozwiązać, proszę, zrób najpierw nowego forka, bo inaczej ciągle nie będe mógl tego automatycznie dodać..
![obraz](https://user-images.githubusercontent.com/5088643/49474713-aae5f900-f815-11e8-9269-dad0a55f6bbf.png)

wrata commented:
Znowu coś pokręciłem. Ja mam nowy plik pyTechBrain3_test.pyw, a tu widzę, że następuje modyfikacja starego pliku.
Coś ten github "mnie nie lubi".
 
 
W dniu 2018-12-04 22:41:05 użytkownik ABIX Adam Jurkiewicz <notifications@github.com> napisał:
Wiesiek, musze to ręcznie rozwiązać, proszę, zrób najpierw nowego forka, bo inaczej ciągle nie będe mógl tego automatycznie dodać..
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
 
 
 
--
Pozdrawiam,
Wiesław Rychlicki

wrata committed:
Program do testowania płytki PyTechBrain

Wymagania programu:
- Python 3.7 
- moduł pymata-aio (https://github.com/MrYsLab/pymata-aio)
- płytka PyTechBrain z oprogramowanie FirmataPlus.ino (https://github.com/MrYsLab/pymata-aio/tree/master/FirmataPlus)

abixadamj reviewed (approved):
OK
